cnf-tests/Dockerfile.openshift

```diff
@@ -58,8 +58,36 @@ WORKDIR $TESTER_PATH
 
 RUN go build -mod=vendor -o /oslat-runner
 
-# build oslat biary
-FROM openshift/origin-release:golang-1.13 as builder-oslat
+# Build cyclictest-runner binary
+FROM golang:1.13 AS builder-cyclictest-runner
```

Comment:
why do you need another builder, which is identical to `builder` above?

Reply:
Done